README.md

```diff
@@ -21,24 +21,17 @@ class Graph {
   Node root;
 }
 class Node {
-  List<Node> children;
+  List<Node> edges;
   // Interesting data
 }
 ```
 
 Any representation can be adapted to the needs of the algorithm:
 
-- Some algorithms need to associate data with each node in the graph and it will
```

Comment:
I think we should add a bullet about the 2 new parameters

Reply:
fixed. PTAL